Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM --platform=$BUILDPLATFORM golang:1.22.5 AS builder
```

Comment:
What were we doing prior to this Docker file?

Reply:
Nothing! There was no image for this CLI. This will be net new (but was proved out in the `saucectl` proof of concept)